concordium-client.cabal

```diff
@@ -37,7 +37,7 @@ flag middleware
 flag static
   description: Use static linking.
   manual: True
-  default: False
+  default: True
```

Comment:
I don't think this should have been changed. Static linking is troublesome during development. Can you change it back to False as the default please.

Reply:
done